includes/Plugin_Context.php

```diff
@@ -67,4 +67,22 @@ public function path( $relative_path = '/' ) {
 	public function url( $relative_path = '/' ) {
 		return plugin_dir_url( $this->main_file ) . ltrim( $relative_path, '/' );
 	}
+
+	/**
+	 * Returns the plugin location.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string The plugin file if single file plugin. Or the plugin folder.
+	 */
+	public function location() {
+		$path = $this->path();
+
+		// Return the plugin path and basename if the path matches the plugin directory.
+		if ( WP_PLUGIN_DIR . '/' === $path ) {
```

Comment:
What about MU plugins?

Reply:
We don't support testing MU plugins at this point, so this is okay as is I think. Testing MU plugins will require lots of additional changes, so let's think about that separately in the future.

Reply:
@jjgrainger Do we needs to open separate issue for MU plugins?

Reply:
Thanks @mukeshpanchal27 @spacedmonkey I've opened an issue to discuss MU Plugin support here https://github.com/10up/plugin-check/issues/154